lib/super_digit.rb

```diff
@@ -1,15 +1,31 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n^2)?
```

Comment:
Since the number of recursive calls depends on the number of digits... this is actually O(log<sub>10</sub>n)